vega_sim/service.py

```diff
@@ -1445,6 +1641,17 @@ def get_latest_market_data(
             asset_decimals_map=self.asset_decimals,
         )
 
+    def get_market_state(
+        self,
+        market_id: str,
+    ) -> str:
+        """
+        Output market state.
+        """
+        return vega_protos.markets.Market.State.Name(
+            self.get_latest_market_data(market_id).market_state
+        )
```

Comment:
Yeah we should be using the enum itself wherever possible. I know we pass it in as strings in a few places (which is more just because of the advantage that the proto can handle that) but we should avoid returning them as strings.

Reply:
@witgaw local enums to translate would probably be the ideal state indeed given that the proto definition isn't handled well by autocomplete, however that would be a pain to manage :D